core/modules/taxonomy/taxonomy.theme.inc

```diff
@@ -25,7 +25,7 @@ function theme_taxonomy_overview_vocabularies($variables) {
 
       $row = array();
       $row[] = backdrop_render($vocabulary['name']);
-      $row[] = backdrop_render($vocabulary['items']);
+      $row[] = array('data' => backdrop_render($vocabulary['items']), 'class' => array('text-align-right'));
```

Comment:
@BWPanda the code is correct, but I have a suggestion re readability (coding standards):

```
      $row[] = array(
        'data' => backdrop_render($vocabulary['items']),
        'class' => array('text-align-right'),
      );
```
That's easier to read than the long one-liner.